Jenkinsfile

```diff
@@ -16,8 +16,7 @@ properties(
       string(name: 'RELEASE_INSTALLER_ONLY', defaultValue: 'false',
         description: """Optional. Set this if we're releasing only the installer."""
       ),
-    ]),
-    disableConcurrentBuilds(),
```

Comment:
this was added in https://github.com/armory-io/k8s-installer/pull/130, but probably can be removed safely